index.html

```diff
@@ -5,8 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
-</head>
+    <link href="styles/index.css" rel="stylesheet" type="text/css"/>
+    <link id="vibe" href="styles/vibe2.css" rel="stylesheet" type="text/css"/>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <<link href="https://fonts.googleapis.com/css2?family=Baloo+Bhaijaan+2&family=Bebas+Neue&family=Cormorant+Garamond:ital@1&family=Fuzzy+Bubbles:wght@700&family=Gwendolyn:wght@700&family=Londrina+Shadow&family=Readex+Pro&display=swap" rel="stylesheet">
 <body>
-    
+    <section class = "container">
+    <header>
+        <h1 id="h1Header">☀️ MyWEATHER ⛈ </h1>
+    </header>
+    <section id="instructions">
+        <section id="skySelect">
+            <select id="sky">
+                <option value="">Select a sky</option>
+                <option value="sunny">Sunny</option>
+                <option value="cloudy">Cloudy</option>
+                <option value="rainy">Rainy</option>
+                <option value="snowy">Snowy</option>
+            </select>
+        </section>
+        <h2 id="inputFormContents">Input your name and location for a weather report made just for you!</h2>
+        <section id="inputFormBox">
+            <h3>Tell me a lil about yourself plz🧞</h3>
+            <form id="inputForm">
+                <label for="fname">Your first name:</label>
+                <input type="text" id="fname" name="fname"><br><br>
```

Comment:
Rather than using `<br>` tags for spacing, prefer setting margins in your style sheets.